site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -28,23 +28,27 @@ To begin our discussion, let's review the initial version of `App`:
 
```

Comment:
@dylans the changes to this tutorial could do with a real review as quite a lot has changed.